processing/src/test/java/io/druid/query/aggregation/post/ArithmeticPostAggregatorTest.java

```diff
@@ -51,7 +51,7 @@ public void testCompute()
     List<PostAggregator> postAggregatorList =
         Lists.newArrayList(
             new ConstantPostAggregator(
-                "roku", 6
+                "roku", 6D
```

Comment:
Expression recognizes this as long type and make long type result, which is different from arithmetic post aggregator which regards all numbers as double.